reconcile/utils/gitlab_api.py

```diff
@@ -707,7 +707,8 @@ def get_project_hooks(self, repo_url):
         if p is None:
             return []
         gitlab_request.labels(integration=INTEGRATION_NAME).inc()
-        return p.hooks.list(per_page=100)
+        # TODO: get_all may send multiple requests, update metrics accordingly
+        return p.hooks.list(per_page=100, get_all=True)
```

Comment:
`per_page` just affect how it fetches items internally, at the end, this method call will return a list with all objects. If there are 150 items, then 2 requests. if `get_all` set to `False` or not set, it will just return the first page.

Reply:
Gotcha, thanks for the explanation!